### ISSUE TITLE
Change sed redirection to in-place editing in docs

### ISSUE THREAD

**Issue by svenstaro** [minishift/minishift#2300]
Currently, the docs advise to do a shell redirection but that won't work with sudo as sudo is not used for the redirection itself. The fix is simple as sed has a built-in in-place editing flag `-i`.

**Comment by anjannath** [comment]
@svenstaro Thanks for reporting, the config file might have changed in the newer versions, when i wrote this there were no white spaces, even now in my system the instructions work, but anyway thanks for reporting this.

Can you please also update the docs with the instructions for defining the default network.

**Comment by svenstaro** [comment]
The default network doesn't have to be defined. It exists after installation.

**Comment by anjannath** [comment]
Last i checked i needed to the additional step of defining it.. maybe it is not needed now.. need to check in my arch box

**Comment by gbraad** [comment]
I wonder if an entry for ArghLinux is really necessary...

**Comment by anjannath** [comment]
@gbraad Yes, people use _Arch Linux_, i know couple of people inside redhat who does. me included.. also 

Many people in Asia does use Arch Linux a lot.. so we should keep it as extra steps are required to make it work.. :-)

**Comment by svenstaro** [comment]
@anjannath By "setting up" the network I meant it needs to be started. I'm _pretty_ sure this fix is enough as I tested it on a clean system and the **default** network was there.

**Comment by gbraad** [comment]
Eh... I am in Asia? And the usage is just a percent. Well, there are people who use ichtus linux, but that does not mean much. it has a limited reach... and besides, wasn't half of the fun of Arch to say you were able to get something working and 'talk' about it ? ;-) Anyway... I still see limited appeal. rather see instructions for SuSE or other more mainstream releases. go ahead and have your fun...

**Comment by anjannath** [comment]
resolved via #2299
